bootstrap/kubeadm/types/upstreamv1beta4/conversion_test.go

```diff
@@ -209,3 +211,10 @@ func hubNodeRegistrationOptionsFuzzer(obj *bootstrapv1.NodeRegistrationOptions,
 		obj.Taints = nil
 	}
 }
+
+func hubClusterConfigurationFuzzer(obj *bootstrapv1.ClusterConfiguration, c randfill.Continue) {
+	c.FillNoCustom(obj)
+
+	obj.CertificateValidityPeriodDays %= 24
+	obj.CACertificateValidityPeriodDays %= 24
```

Comment:
I assume this avoids an overflow in metav1.Duration. Can we use this instead?
```go
	obj.CertificateValidityPeriodDays = c.Int31n(3 * 365 + 1)
	obj.CACertificateValidityPeriodDays = c.Int31n(100 * 365 + 1)
```
This limits the days to our supported min/max range